docs/rules/no-return-await.md

```diff
@@ -36,6 +32,10 @@ async function foo() {
         return await bar();
     } catch (error) {}
 }
+
+async function foo() {
+    return bar();
+}
 ```
 
 In the last example the `await` is necessary to be able to catch errors thrown from `bar()`.
```

Comment:
A problem with the change is that this sentence now does not apply to the 'last' example. I'm not sure what would be the best way to fix this.

Reply:
Ah yeah, good catch. Okay, the value of this change is officially not worth the trouble it has created for you guys. Closing it, thanks for your time :)